README.md

```
@@ -100,6 +134,153 @@ $logs = $audit->getLogsByResource(
 ); // Returns an array of all logs for the specific resource
 ```
 
+**Batch Logging**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)